cmd/bootstrap/runner.go

```diff
@@ -511,3 +561,42 @@ func (r *runner) installOperator(ctx context.Context, helmClient helmclient.Inte
 
 	return nil
 }
+
+func (r *runner) patchChartOperatorDeployment(ctx context.Context, k8sClients k8sclient.Interface) error {
+	labelSelector := "app.kubernetes.io/instance=chart-operator-unique"
+	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("waiting for chart-operator with label selector %#q", labelSelector))
+
+	o := func() error {
+		list, err := k8sClients.K8sClient().AppsV1().Deployments(namespace).List(ctx, metav1.ListOptions{LabelSelector: labelSelector})
+
+		if apierrors.IsNotFound(err) || len(list.Items) != 1 {
+			r.logger.LogCtx(ctx, "level", "debug", "message", "chart-operator deployment not ready yet")
```

Comment:
```suggestion
			r.logger.LogCtx(ctx, "level", "debug", "message", "chart-operator deployment not created yet")
```

The not ready confused me a bit. As we don't need to wait for the pod to be ready.